genesis-tools/genesis-populate/src/lib.rs

```diff
@@ -227,7 +226,7 @@ impl GenesisBuilder {
 
     fn add_additional_account(&mut self, account_id: AccountId) -> Result<()> {
         const TESTING_INIT_BALANCE: Balance = 1_000_000_000_000_000;
-        const TESTING_INIT_STAKE: Balance = 50_000_000;
+        const TESTING_INIT_STAKE: Balance = 0;
```

Comment:
will non zero init_stake hurt something. If so I'll rerun

Reply:
Better re-run.

Reply:
There are some operations that might cause the system to freak out, since we did not process staking properly, but just added stake to the state dump.